NEWS

```diff
@@ -29,6 +29,28 @@ New Functionality
 
 - The new ``is_valid_subnet()`` function mirrors ``is_valid_ip()``, for subnets.
 
+- A new Storage framework was merged into the Zeek tree. The intention with this framework
+  is to eventually completely replace the storage functionality that Broker provides,
+  including direct storage via calls such as ``Cluster::create_store`` and
+  ``Broker::put_unique`` as well as storage-backed tables via the ``&backend``
+  attribute. This is an initial version for testing, and will be expanded upon in the
+  future. The current state of the framework is as follows:
+
+  - Functions for opening and closing backends, and insertion, retrieval, and erasure of
+    elements are available.
+
+  - Backends can support both asynchronous mode (using ``when`` statements) and
+    synchronous mode (blocking until the operation copmletes). BIF methods were added
+    under new ``Storage::Async`` and ``Storage::Sync`` namespaces for these two modes. The
+    modes can be used interchangeably with the same backend handle.
+
+  - SQLite and Redis backends exist in the Zeek tree by default. We are working on a
+    backend for NATS that will be available as an external plugin, but it is not quite
+    ready yet. Both of the existing backends support usage in a cluster environment.
```

Comment:
I wouldn't mention NATS at all here. It could be more interesting to mention that alternative backends can be provided via plugins as that isn't quite mentioned.